### ISSUE TITLE
Relationship attributes not being returned along with included members.

### ISSUE THREAD

**Issue by dhirajdhakal09** [joukevandermaas/saule#113]
I could not generate the output with attributes in the relationship node, only 'link' is returned and also included members are not displayed.  Please help me asap.
		
		This is how i defined my controller 
		
		[HttpGet]
        [ReturnsResource(typeof(NumberResource))]
        [Route("api/numbers")]
        public IHttpActionResult GetSomeNumbers()
        {
            return Ok(MockupGetSomeNumbers());
        }

        public IQueryable MockupGetSomeNumbers()
        {
            var myNumber = new Number
            {
                Id = new Guid(),
                Number = "12345",
                Plan = new Plan
                {
                    Id = new Guid(),
                    Description = "Some Description"
                }
            };
            var someQueryableNumbers = new List<Number>();
            someQueryableNumbers.Add(myNumber);
            return someQueryableNumbers.AsQueryable();
        }
		
		And the following is my resources
		
		public class NumberResource : ApiResource
		{
			public NumberResource()
			{
				OfType("numbers");
				Attribute("Number");

					  
				BelongsTo<PlanResource>("plans");
			}
		}

		public class PlanResource : ApiResource
		{
			public PlanResource()
			{
				OfType("plans");
				Attribute("Description");
			}

		}
		
		And the following is the output I got
		
		{
			"data": [
			{
			"type": "numbers",
			"id": "00000000-0000-0000-0000-000000000000",
			"attributes": {
			"number": "12345",
	
			  },
			  "relationships": {

				"plans": {
				  "links": {
					"self": "http://localhost:58739/api/numbers/00000000-0000-0000-0000-000000000000/relationships/plans/",
					"related": "http://localhost:58739/api/numbers/00000000-0000-0000-0000-000000000000/plans/"
				  }
				}
			  },
			  "links": {
				"self": "http://localhost:58739/api/numbers/00000000-0000-0000-0000-000000000000/"
			  }
			}
			],
			"links": {
			"self": "http://localhost:58739/api/numbers"
			}
		}

**Comment by joukevandermaas** [comment]
When including relationships, there are three options:

1. **The property does not exist**  
  Saule will generate a link according to [the spec](http://jsonapi.org/format/#document-links).
2. **The property exists, but it is null**  
  Saule will generate a link.
3. **The property exists and is not null**  
  Saule will add the resource to the `included` hash according to [the spec](http://jsonapi.org/format/#document-compound-documents).

In your case, you need to set the value of a `Plan` property on `Number`.

**Comment by dhirajdhakal09** [comment]
Hi,

I have already set the value of plan property on number, I showed in
the example too, doing like as follows:

  var myNumber = new Number
        {
            Id = new Guid(),
            Number = "12345",
            Plan = new Plan
            {
                Id = new Guid(),
                Description = "Some Description"
            }
        };

But still attribute of Plan resource not seen in the Number resource.

Moreover, I tried to use the same resource provided in the wiki, even
on doing that I cannot see attribute in relationship.

I am using Visual Studio 13 Community Edition.

Thanks!!
